Documentation/CrossPlatform/LeapMotionMRTK.md

```diff
@@ -0,0 +1,117 @@
+# How to Configure Leap Motion Hand Tracking in MRTK 
+
+A [Leap Motion Controller](https://www.ultraleap.com/product/leap-motion-controller/) is required to use this data provider.
+
+The Leap Motion Data Provider enables articulated hand tracking for VR and could be useful for rapid prototyping in the editor.  The data provider can be configured to use the leap motion controller mounted on a headset or placed on a desk face up.
+
+## Using Leap Motion tracking in MRTK
+1. Prepare MRTK project for Leap Motion
+
+    - This step only applies if the source of MRTK is cloned from the git repo and NOT the unity packages.
+
+        - Navigate to **Mixed Reality Toolkit > Utilities > Update > Configure CSC File for Leap Motion**. Updating the csc file filters out the obsolete warnings produced by the Leap Motion Core Assets.
```

Comment:
For these I like to usually also point to some more details for what's going on - we dont' have to write out fully here why this is necessary, but even a link to a comment on an issue explaining what this step is for is kinda nice to have.

Reply:
This issue has been filed in the leap repo here https://github.com/leapmotion/UnityModules/issues/1082

Added more details